packages/core/test/stackViewport_gpu_render_test.js

```diff
@@ -778,7 +778,10 @@ describe('renderingCore -- Stack', () => {
       });
     });
 
-    it('Should be able to calibrate an image', function (done) {
+    const skipIt = () => null;
+    // TODO - renable this when affine transforms are supported as part of
+    // the calibration event instead of simple calibration ratios
+    skipIt('Should be able to calibrate an image', function (done) {
```

Comment:
why skipit

Reply:
Because they didn't implement it.skip(....) and I wanted the code to be compiled but not executed right now.  It is a test that we probably want in the future.

Reply:
can't you comment it out then?

Reply:
I wanted it to continue compiling, and if you comment it out, then it stops being updated/fixed when the API changes.